main/index.js

```diff
@@ -1,15 +1,31 @@
 'use strict'
 
+const { app, dialog } = require('electron')
+const log = require('electron-log')
+const path = require('node:path')
+
+const inTest = (process.env.NODE_ENV === 'test')
+const isDev = !app.isPackaged && !inTest
+
+// Override the place where we look for config files when running the end-to-end test suite.
+// We must call this early on, before any of our modules accesses the config store.
+// https://www.npmjs.com/package/electron-store
+// https://www.electronjs.org/docs/latest/api/app#appgetpathname
+if (inTest && process.env.HOME) {
+  app.setPath('userData', path.join(process.env.HOME, 'user-data'))
```

Comment:
also here, what about putting it in `/tmp` instead? And shouldn't it be a new directory every time?

We could use `join(os.tmpdir(), crypto.randomUUID())`